### ISSUE TITLE
There is no way to set API_ENDPOINT to secure HTTPS

### ISSUE THREAD

**Issue by villuc2c** [transloadit/ruby-sdk#58]
https://github.com/transloadit/ruby-sdk/blob/42ad4585fb63da7761b911d3b55a9757dd586100/lib/transloadit/request.rb#L123-L128
The request goes out as insecure HTTP with no way of setting as secure

**Comment by rmehner** [comment]
The default API Endpoint is HTTPS already: https://github.com/transloadit/ruby-sdk/blob/main/lib/transloadit/request.rb#L13

**Comment by villuc2c** [comment]
Try this
```rb
URI.parse("https://api2.transloadit.com/").host
```
and show me how the HTTPS is passed to the RestClient::Resource.new
or from a different angle

So I have a page, that responds to HTTP and HTTPS, but doesn't have the proper certs
```sh
curl https://lehe.ga
curl http://lehe.ga
```
So calling a https will fail, but http will not

```rb
api = URI.parse('https://lehe.ga')
RestClient::Resource.new(api.host).get.body
RestClient::Resource.new(api.to_s).get.body
```

**Comment by rmehner** [comment]
Huh, very good catch. Definitely needs to be fixed (and looks like an rather easy fix too). Thanks for reporting!

**Comment by villuc2c** [comment]
Would have been a good possible bug-bounty due to this being a default behaviour for all that use ruby-sdk, allowing all middle servers to intercept keys and secrets and maybe more 😄 

Anyway, glad I could help

**Comment by Acconut** [comment]
This is fixed in #64.
